### PR TITLE
Dump payload when we are writing to file.

### PR DESCRIPTION
**Pull request by jondeandres**

### CODE REVIEW

File: lib/rollbar.rb [hunk 1]
```diff
@@ -517,7 +517,7 @@ def do_write_payload(payload)
           @file = File.open(configuration.filepath, "a")
         end
 
-        @file.puts payload
+        @file.puts(dump_payload(payload))
```
Comment (brianr) [inline lib/rollbar.rb]: I kinda think that we should do this first, before opening the file, to make things simpler in case the dump fails. Thoughts?

Comment (jondeandres) [inline lib/rollbar.rb]: You are right @brianr, I've just changed it.

**Comment by jondeandres** [comment]
@brianr tests are passing here, I think we can merge this and release a new version.

**Comment by brianr** [comment]
Released as 1.3.2

**Pull request merged**